allocate.go

```diff
@@ -338,6 +338,7 @@ func findExecPath() string {
 		"chrome.exe", // in case PATHEXT is misconfigured
 		`C:\Program Files (x86)\Google\Chrome\Application\chrome.exe`,
 		`C:\Program Files\Google\Chrome\Application\chrome.exe`,
+		os.Getenv("USERPROFILE") + `AppData\Local\Google\Chrome\Application\chrome.exe`,
```

Comment:
are we sure that `USERPROFILE` will end with a path separator? it feels like `filepath.Join` would be safer here

Reply:
Oops, you're totally right.